### DISCUSSION TITLE
Open DTU "Inverter Gesamtertrag heute" = DC Ertrag?

### DISCUSSION THREAD

**Discussion by Tomv888** [tbnobody/OpenDTU#1905] [General]
Von den einzelnen Strings werden die Tageserträge angezeigt,also das was das Modul DC-seitig erzeugt hat.
Rechnet man die Strings alle zusammen,kommt man auf das,was im Feld "Allgemein" unter Tagesertrag steht
und das ist der gleiche Wert wie oben unter "Inverter Gesamtertrag heute"

"Inverter Gesamtertrag heute" scheint also der DC Ertrag und nicht der AC Ertrag zu sein.
Ist dann der Wert bei "Inverter Gesamtertrag Insgesamt" ebenfalls der DC Ertrag? Kann der AC Ertrag nicht ausgelesen werden?

**Comment by Tomv888** [comment]
Ist meine Frage nicht zu beantworten?
So schwer?

**Comment by Scootch2002** [comment]
Ich kenne keinen WR (oder Monitoring) der den DC Ertrag ausgibt. Sinn macht ja nur der AC Ertrag.

**Reply by Tomv888** [reply]
Völlig richtig,jedoch ist es so,das der unter den Strings angezeigte Tagesertrag der DC Ertrag ist (weil String immer DC ist).nimmt man alle Strings in Summe,stimmt der Wert mit dem unter "Inverter Gesamtertrag Heute" gezeigte Wert überein.

**Reply by NintendoTito** [reply]
Ich kann deine Frage und deine Berechnung nachvollziehen. Aber woher nimmst du die Info, dass der bei den einzelnen Modulen angezeigte Tagesertrag zwangsläufig der DC Ertrag ist? Am Tagesertrag steht keine detaillierte Einheit mit dran.

**Reply by NintendoTito** [reply]
Ich denke es ist ähnlich wie bei der Leistung. Wenn nix näheres dabei steht, dann ist die AC Leistung, also die, die für den Endnutzer "brauchbar" ist. Die Leistung DC ist auch extra als solche ausgewiesen. Wenn der Tagesertrag als DC wäre, dann würde es vermutlich auch mit dabei stehen.

**Comment by Tomv888** [comment]
ja da könntest du Recht haben.Ist halt nur verwirrend, das im String Feld ein AC Ertrag steht
